quesma/optimize/cache_group_by.go

```diff
@@ -0,0 +1,37 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package optimize
+
+import "quesma/model"
+
+// cacheGroupByQueries - a transformer that suggests db to cache the query results
+//
+// It's done by adding settings to the query
+//
+// https://clickhouse.com/docs/en/operations/query-cache
+//
+// Cached queries can be examined with:
+//
+// select * from system.query_cache
+//
+// Cache can be dropped with
+//
+//  SYSTEM DROP QUERY CACHE
+//
+
+type cacheGroupByQueries struct {
+}
+
+func (s *cacheGroupByQueries) Transform(queries []*model.Query) ([]*model.Query, error) {
+
+	for _, query := range queries {
+
```

Comment:
FIY: It would be nice if customer could write hints like that.

Reply:
Just like any other transformer.